### DISCUSSION TITLE
Outage issues are not creating - what's going on?

### DISCUSSION THREAD

**Discussion by Hans5958** [upptime/upptime#288] [Q&A] [answered]
Hello, everyone. I just created a new Upptime instance to monitor a site that I expect to have a lot of downtimes. The instance figures out that some are down, but it doesn't create any issue at all.

After investigating the issue, it seems that it all comes down to a part of the code that tried to obtain the issues that are currently open. Here's an example.

```
Checking https://ltmpt.ac.id
Current status undefined up 2021-03-15T04:28:17.000Z
Result from test 502 4.026185
Result from test 502 0.23057
Result from test 502 10.775742
[master fefbe07] 🟥 Situs Utama LTMPT is down (502 in 4026 ms) [skip ci] [upptime]
 1 file changed, 4 insertions(+), 4 deletions(-)
fefbe076f0e84cbe7f80f5038a93e5265775496c
Status is different up to down
ERROR RequestError [HttpError]: Not Found
    at /home/runner/work/_actions/upptime/uptime-monitor/v1.26.1/node_modules/@octokit/request/dist-node/index.js:66:1
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at Object.update (/home/runner/work/_actions/upptime/uptime-monitor/v1.26.1/dist/update.js:220:1) {
  name: 'HttpError',
  status: 404,
  headers: {
    'access-control-allow-origin': '*',
    'access-control-expose-headers': 'ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, Deprecation, Sunset',
    connection: 'close',
    'content-encoding': 'gzip',
    'content-security-policy': "default-src 'none'",
    'content-type': 'application/json; charset=utf-8',
    date: 'Mon, 15 Mar 2021 08:30:10 GMT',
    'referrer-policy': 'origin-when-cross-origin, strict-origin-when-cross-origin',
    server: 'GitHub.com',
    'strict-transport-security': 'max-age=31536000; includeSubdomains; preload',
    'transfer-encoding': 'chunked',
    vary: 'Accept-Encoding, Accept, X-Requested-With',
    'x-content-type-options': 'nosniff',
    'x-frame-options': 'deny',
    'x-github-media-type': 'github.v3; format=json',
    'x-github-request-id': '0780:21E8:20D18EF:362AE9C:604F1B12',
    'x-ratelimit-limit': '1000',
    'x-ratelimit-remaining': '950',
    'x-ratelimit-reset': '1615797886',
    'x-ratelimit-used': '50',
    'x-xss-protection': '1; mode=block'
  },
  request: {
    method: 'GET',
    url: 'https://api.github.com/issues?owner=Hans5958&repo=LTMPT-Upptime&labels=situs-utama-ltmpt&filter=all&state=open&sort=created&direction=desc&per_page=1',
    headers: {
      accept: 'application/vnd.github.v3+json',
      'user-agent': 'Auto5958 octokit-rest.js/17.11.2 octokit-core.js/2.5.4 Node.js/12.13.1 (Linux 5.4; x64)',
      authorization: 'token [REDACTED]'
    },
    request: { hook: [Function: bound bound register] }
  },
  documentation_url: 'https://docs.github.com/rest/reference/issues#list-issues-assigned-to-the-authenticated-user'
}
Checking https://portal.ltmpt.ac.id
Current status undefined down 2021-03-15T04:28:20.000Z
Result from test 200 1.960745
Result from test 200 2.262733
Result from test 200 1.339126
Skipping commit,  status is down
```
https://github.com/Hans5958/LTMPT-Upptime/runs/2110870348?check_suite_focus=true

Similar logs can be found on these links below.
- https://github.com/Hans5958/LTMPT-Upptime/runs/2110773292?check_suite_focus=true
- https://github.com/Hans5958/LTMPT-Upptime/runs/2111001234?check_suite_focus=true

I'm not sure what causes this. Tokens have been added, repo name has been configured, permissions have been given, but the issues are not creating. What is going on?

I hope you guys can help me solve this issue. H

**Comment by rodenacker** [comment]
Hi
My issues are also not being created. The status of my broken site is 'down', but the uptime stats say 100% uptime. 
I have checked the setup again, but can't find any problems there. 
https://rodenacker.github.io/linx-upptime/
Please help.
Thanks, Franz

**Reply by thomasmerz** [reply]
Although it seems that I had mixed up my secret URLs in my repository secrets, but @upptime-bot also found some downtimes during my planned maintenance/reboots:  
* One services in datacenter in HEL was down, rebooted and up again - upptime did a great job: it recognized the downtime, but didn't create an issue due to the very short duration 👍🏻 
* Another services in datacenter in NGB was down, rebooted and still down (until I found out by accident!) - upptime recognized the downtime (as you can see on the screnshot https://user-images.githubusercontent.com/18568381/111387067-71b3b900-86ad-11eb-8967-ff5f571104c2.png but upptime didn't create an issue for this really long lasting downtime 😞 

I documented this in my repo before I found this discussion: https://github.com/thomasmerz/upptime/issues/94

**Reply by ronadevelopment** [reply]
Unfortunately I am experiencing the same problem. Please let me know if someone finds a solution for this problem.

**Reply by thomasmerz** [reply]
@AnandChowdhary , there are potentially all users of upptime affected regarding no issues being created for any "down" service 😢 Could please give us an update and an assessment of the situation? Many thanks.

**Reply by AnandChowdhary** [reply]
This has been fixed and I'm waiting to see the update now.

**Reply by ronadevelopment** [reply]
For me this error is still occuring.

**Comment by Hans5958** [comment]
UPDATE: It seems that it might caused because it uses the GITHUB_TOKEN instead of the GH_PAT that I supplied. I think @AnandChowdhary has noticed this way before me and has pushed a fix. 

I have forced it so it uses GH_PAT, but it does not work. I forced the template update, and it still doesn't work. Weirdly is that when the Setup CI is run, it reverts back to GITHUB_TOKEN, and I need to force it so it uses GH_PAT and update the template. Probably because it takes the latest version instead of the master revision, so I need to wait another version to be released? Hopefully it gets fixed soon.

EDIT: I have also created https://github.com/Hans5958/PreMiD-Upptime-Next. No issues are creating there also.

**Reply by thomasmerz** [reply]
@Hans5958 , why and where exactly do you see a correlation to #287 ? I'm uptodate, but still no issues are created for my down services 😢

**Reply by Hans5958** [reply]
I have no dang clue. It's just my theory.

**Reply by leahcimto** [reply]
Issues are still not being created for me.

**Comment by AnandChowdhary** [comment, marked as answer]
The fix has been released, but if it's not in your repository, please do this manually:

1. Find and replace `GITHUB_TOKEN` with `GH_PAT` in your `.github/workflows/update-template.yml`
2. Run Update Template CI [manually](https://upptime.js.org/docs/triggers#manual-triggers)
3. This should auto-update your Upptime to the latest version, which both fixes the updating problem and now throws an error in case the update couldn't be pushed.
